lib/rucio/client/downloadclient.py

```diff
@@ -1089,3 +1090,15 @@ def _check_output(self, output_items):
             raise NotAllFilesDownloaded()
 
         return output_items
+
+    def _send_trace(self, trace):
+        """
+        Checks if sending trace is allowed and send the trace.
+
+        :param trace: the trace
+        """
+        logger = self.logger
+        if self.tracing:
+            send_trace(trace, self.client.host, self.client.user_agent)
+        else:
+            logger.info('Tracing was turned off.')
```

Comment:
i would have this as debug only

Reply:
I would remove this completely at this point at output it just once in init()